setup.py

```diff
@@ -52,7 +52,7 @@
     include_package_data=True,
     classifiers=[
         "Development Status :: 4 - Beta",
-        "License :: OSI Approved :: Apache Software License",
+        "Apache 2.0",
```

Comment:
If we want to fix the license declarations, we should probably upgrade to the SPDX License Expressions (https://spdx.org/licenses/Apache-2.0.html) as also done in https://peps.python.org/pep-0639/, which basically deprecates this usage in classifiers: https://peps.python.org/pep-0639/#deprecate-license-classifiers